app/services/invite_schools.rb

```diff
@@ -366,26 +374,19 @@ def find_school(urn)
   end
 
   def create_and_send_nomination_email(email, school)
-    nomination_email = NominationEmail.create_nomination_email(
-      sent_at: Time.zone.now,
-      sent_to: email,
+    access_token = SchoolAccessToken.create!(
       school: school,
+      permitted_actions: %i[nominate_tutor],
     )
-    send_nomination_email(nomination_email)
-  rescue Notifications::Client::RateLimitError
-    sleep(1)
-    send_nomination_email(nomination_email)
+    send_nomination_email(to: email, access_token: access_token)
   end
 
-  def send_nomination_email(nomination_email)
-    notify_id = SchoolMailer.nomination_email(
-      recipient: nomination_email.sent_to,
-      school: nomination_email.school,
-      nomination_url: nomination_email.nomination_url,
-      expiry_date: email_expiry_date,
-    ).deliver_now.delivery_method.response.id
-
-    nomination_email.update!(notify_id: notify_id)
+  def send_nomination_email(to:, access_token:)
+    SchoolMailer.nomination_email(
+      recipient: to,
+      school: access_token.school,
+      access_token: access_token,
+    ).deliver_later
```

Comment:
Just a thought - could the creation of the access token be moved into the school mailer to clean the method calls up a bit as it seems to be passed around a bit? Also is there a risk here that if background jobs are blocked for some reason the access_token could expire before being sent? (I'm guessing that's highly unlikely)